Files/Filesystem/ListedItem.cs

```diff
@@ -99,6 +99,12 @@ public BitmapImage FileImage
             }
         }
 
+        bool isItemPinnedToStart;
+        public bool IsItemPinnedToStart {
```

Comment:
Change to `public bool IsItemPinnedToStart => App.SecondaryTileHelper.CheckFolderPinned(itemPath),` like the IsPinned property

Reply:
@gave92 Good idea